### DISCUSSION TITLE
Heads up regarding step 3 and opening the Completions Panel

### DISCUSSION THREAD

**Discussion by tecrogue** [skills/.github#536] [Code with Copilot]
It looks like the newest version of the Copilot extension available in Codespaces as of this comment (1.176.0) does not include the function for the Completions Panel, either by the dropdown menu or Control+Enter. 

If you are getting an error message "command 'github.copilot.generate' not found" when you try to bring it up that is the cause.

If you change the version of the extension to 1.175.0 it will work via the keyboard shortcut.

**Comment by Masei25** [comment]
Please how do I change the extension, I am having the same issue

**Reply by tecrogue** [reply]
If you go to settings (the gear in the bottom left, select Extensions, then under GitHub Copilot click the gear, the select Install `Another Version...` it will let you select the version of the extension that is used in that codespace.
<img width="551" alt="image" src="https://github.com/skills/.github/assets/5312877/8fc63b09-c5ca-45bf-a732-b0ce448eb758">

**Reply by Masei25** [reply]
Thank you, I was able to change my version to 1.175.0 but the completion panel isn't still coming up

**Reply by tecrogue** [reply]
I just tried it on the current version, 1.180.0, and it came right up with control+enter so you should be fine with the current version instead. What method are you trying to use to bring up the completion panel?

**Comment by Masei25** [comment]
I have tried updating to the latest version as well (1.180.0), I used the control+enter but I got 'No suggestion', not sure what the issue might be.
![snipppp](https://github.com/skills/.github/assets/33876391/14650884-d842-485a-b0a0-2963f31deae3)
